facia/public/facebook_ia_remote.html

```diff
@@ -0,0 +1,172 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="robots" content="noindex">
+    <script>
+    </script>
+</head>
+<body style="margin:0">
+<div id="c" style="position:absolute;top:0;left:0;bottom:0;right:0;">
+
+    <script>
+        (function () {
```

Comment:
Apologies, I missed this wrapping function the first time I went through. Either you can do what @cb372 said and use `window.config`, or make it a global variable by not wrapping it in this function here.